tests/playwright-test/web-server.spec.ts

```diff
@@ -426,6 +426,7 @@ test(`should support self signed certificate`, async ({ runInlineTest, httpsServ
     'playwright.config.js': `
       module.exports = {
         webServer: {
+          command: 'unused',
```

Comment:
I wonder whether this change could break users in the wild, in the same way as this test breaks. Some examples: https://github.com/search?type=code&q=%2Fcommand%3A+%5C%27%5C%27%2F+path%3Aplaywright.